src/main/java/com/vivosys/osgi/deps/OsgiDepsActivator.java

```diff
@@ -49,4 +60,20 @@ public void start(BundleContext bundleContext) throws Exception {
     public void stop(BundleContext bundleContext) throws Exception {
     }
 
+    private static int loadMinStartLevel() {
+        String minStartLevel = System.getProperty(MIN_START_LEVEL_PROPERTY);
+        Integer retVal = DEFAULT_MIN_START_LEVEL;
+        
+        logger.info("Loading property {} to specify minimal relevant start level for OSGi deps graph", MIN_START_LEVEL_PROPERTY);
+        
+        if (minStartLevel != null) {
+            try {
+                retVal = Integer.parseInt(minStartLevel);
+            } catch (NumberFormatException  e) {
+                logger.error("Property {} not in numeric format: {}", MIN_START_LEVEL_PROPERTY, minStartLevel);
+            }
+        }
+        logger.info("Building deps graph for start levels {} or above", retVal);
+        return retVal;      
```

Comment:
Thanks for your contribution!

I prefer a coding style without the single-return `retVal` in general, see https://softwareengineering.stackexchange.com/questions/118703/where-did-the-notion-of-one-return-only-come-from. Please update accordingly.